__tests__/lib/http/proxy-agent.js

```diff
@@ -19,6 +27,10 @@ describe( 'validate CreateProxyAgent', () => {
 		}
 	} );
 
+	afterAll( () => {
+		process.env = { ...env };
+	} );
```

Comment:
Maybe this is unnecessary, but I prefer restoring things to their original state.